op-batcher/batcher/service.go

```diff
@@ -41,7 +41,8 @@ type BatcherConfig struct {
 
 	// UsePlasma is true if the rollup config has a DA challenge address so the batcher
 	// will post inputs to the Plasma DA server and post commitments to blobs or calldata.
-	UsePlasma bool
+	UsePlasma             bool
+	CheckRecentTxsOnStart bool
```

Comment:
See other comment at CLI config, we should also add a field `WaitNodeSync bool`. I also suggest we keep the field CLI config names, so
```suggestion
	
	CheckRecentTxs bool
	WaitNodeSync   bool
```

It would also be more readable to not group them with `UsePlasma`, but instead either in their own group, or above with the others networking configuration fields.